common/services/prismic/link-resolver.test.ts

```diff
@@ -1,27 +1,149 @@
-import linkResolver from './link-resolver';
-
-/**
- * articles and webcomics look and act the same, but we're unable
- * to change the types of documents in Prismic.
- * {@link} https://community.prismic.io/t/import-export-change-type-of-imported-document/7814
- */
-describe('webcomic edge case', () => {
-  test.each([
-    { doc: { type: 'webcomics', uid: '1' }, path: '/stories/1' },
-    { doc: { type: 'webcomic-series', uid: '1' }, path: '/series/1' },
-  ])('$doc resolves to $path', ({ doc, path }) => {
-    expect(linkResolver(doc)).toBe(path);
-  });
-});
-
-it('resolves exhibition guides to /guides/exhibitions/{id}', () => {
-  const doc = { type: 'exhibition-guides', uid: '1' };
-  expect(linkResolver(doc)).toBe('/guides/exhibitions/1');
-});
+import linkResolver, { HighlightTourType } from './link-resolver';
 
+// We want to be able to pass in either Prismic or Content API content types
 test.each([
-  { doc: { type: 'articles', uid: '1' }, path: '/stories/1' },
+  // if ('tags' in doc) { and pages
+  // visual storys
+  { doc: { type: 'articles', uid: '1' }, path: '/stories/1' }, // Prismic type
+  { doc: { type: 'Article', uid: '1' }, path: '/stories/1' }, // Content API type
+  { doc: { type: 'books', uid: '1' }, path: '/books/1' },
+  { doc: { type: 'Book', uid: '1' }, path: '/books/1' },
+  { doc: { type: 'events', uid: '1' }, path: '/events/1' },
+  { doc: { type: 'Event', uid: '1' }, path: '/events/1' },
+  { doc: { type: 'event-series', uid: '1' }, path: '/event-series/1' },
+  { doc: { type: 'Event series', uid: '1' }, path: '/event-series/1' },
+  { doc: { type: 'exhibitions', uid: '1' }, path: '/exhibitions/1' },
+  {
+    doc: { type: 'exhibition-highlight-tours', uid: '1' },
+    path: '/guides/exhibitions/1',
+  },
+  {
+    doc: { type: 'Exhibition highlight tour', uid: '1' },
+    path: '/guides/exhibitions/1',
+  },
+  // With highlightTourTypes
+  {
+    doc: {
+      type: 'Exhibition highlight tour',
+      uid: '1',
+      highlightTourType: 'text' as HighlightTourType,
```

Comment:
is this a thing? I _think_ highlightTourType can only be one of `bsl` or `audio` because those share the same data. `Exhibition text` is its own type I think?

Reply:
It's for handling the legacy exhibition-guides type